modules/installation-complete-user-infra.adoc

```diff
@@ -192,7 +192,13 @@ $ oc get nodes -w
 ====
 If you have additional machine types such as infrastructure nodes, repeat the process for these types.
 ====
-endif::ibm-z,ibm-power[]
+endif::ibm-power[]
+
+ifdef::ibm-z[]
+. Additional steps are required to enable multipathing. Do not enable multipathing during installation.
```

Comment:
@bobfuru I just realized I accidentally dropped the FCP part.  

s/Additional steps are required to enable multipathing/For an installation with Fibre Channel Protocol (FCP), additional steps are required to enable multipathing.